pkg/kms/creator.go

```diff
@@ -45,21 +42,11 @@ func NewServiceCreator(c *Config) ServiceCreator {
 		keystoreID := mux.Vars(req)[keystoreIDQueryParam]
 		keyURI := fmt.Sprintf(masterKeyURI, keystoreID)
 
-		b, err := cloneRequestBody(req)
-		if err != nil {
-			return nil, err
-		}
-
-		p := struct {
-			Passphrase string `json:"passphrase"`
-		}{}
+		// TODO(#22): Replace with split shares. Consider to use "Hub-Kms-Secret-A" and "Hub-Kms-Secret-B-Path"
```

Comment:
We should think about whether it is the server or client that combines the split shares.

Reply:
Ok. In our latest design, we decided that Key Server will perform combining, but we can come back to that question when working on https://github.com/trustbloc/hub-kms/issues/22.